### PR TITLE
Update Specification for "Omni Protocol" rename

### PR DESCRIPTION
**Pull request by msgilligan**
THIS PULL REQUEST IS NOW READY TO MERGE.

The first commit only changes the Title and the Terminology section. Once we have consensus on these changes, I’ll update the remainder of the document.

We'll keep this Pull Request open until we have consensus on Title/Terminology, have updated the entire document to the new terminology, and have had a final review. 

Subsequent commits have changed all instances of Master/Mastercoin to Omni except where they are references to the historical Mastercoin or to the actual Mastercoin/Mastercoins tokens.

### CODE REVIEW

**Comment by dexX7** [comment]
It's really, really nice to see some movement here. :)

The changes are fine in my opinion.

**Comment by zathras-crypto** [comment]
Looking good so far :)

**Comment by msgilligan** [comment]
OK, all instances of *Master Protocol* have been changed to *Omni Protocol* and in many places *Master* or *Mastercoin* has become *Omni*. No changes where made when referring to:

1. *Mastercoin*/*Mastercoins* the currency/currency-unit
1. *Mastercoin Ecosystem* or *Test Mastercoin Ecosystem*
1. *Mastercoin Foundation* (which continues to exist as a separate entity from the *Omni Foundation*)

Unless there are any major objections or errors, I recommend **accepting** this pull request and implementing any further changes (e.g. referring to tokens as *Omnis*) in one or more subsequent pull requests.

**Comment by msgilligan** [comment]
Note: The date field in the revision history needs updating -- I'll do that assuming I get positive feedback at today's All-hands Meeting.

File: README.md [hunk 1]
```diff
@@ -343,8 +351,10 @@ This section defines the fields that are used to construct transaction messages.
     *   11: [Mark a Savings Address as Compromised](#marking-a-savings-address-as-compromised)
     *   12: [Mark an Address as Rate-Limited](#marking-an-address-as-rate-limited)
     *   14: [Remove a Rate Limitation](#removing-a-rate-limitation)
-    *   30: [Register a Data Stream](#registering-a-data-stream)
-    *   31: [Publish Data](#publishing-data)
+    *   30: [Register a Data Stream](#registering-a-data-stream)
+
```
Comment (dexX7) [inline README.md]: No need to insert a blank line here imho.

File: README.md [hunk 2]
```diff
@@ -343,8 +351,10 @@ This section defines the fields that are used to construct transaction messages.
...
+    *   31: [Publish Data](#publishing-data)
+
```
Comment (dexX7) [inline README.md]: Here neither.

File: README.md [hunk 3]
```diff
@@ -912,11 +924,15 @@ Say that you wanted to transfer the Issuer on Record status to another address o
 
 # Future Transactions
 
-The transactions below are still subject to revision and therefore are not included in deployments based on this version of the spec. 
-
-## Creating a List of Addresses
-
-The Master Protocol allows the creation of a list of addresses which can then be referenced by other transactions. For instance, some tokens may be restricted to only be used by a set of approved addresses, such as addresses of people who have provided identifying documentation in compliance with KYC (know your customer) AML (anti-money-laundering) laws. See the introduction to [Smart Property](#smart-property) above for details on how to restrict a token to a set of addresses.
+The transactions below are still subject to revision and therefore are not included in deployments based on this version of the spec. 
+
```
Comment (dexX7) [inline README.md]: Too many blank lines

**Comment by dexX7** [comment]
Looks good.

> I recommend accepting this pull request

Who is currently the maintainer of the spec?

> Note: The date field in the revision history needs updating

In my opinion this is a good time to ditch the history, and switch to a release/tag model. Related: https://github.com/OmniLayer/spec/issues/188

File: README.md (hunk 3 above)
Comment (msgilligan) [inline README.md]: I think the original file may have had a mixture of CR/LF and LF-based newlines which confused my editor. It should be consistently LF-based now.

**Comment by msgilligan** [comment]
Thanks, @dexX7.

I don't think there is an official *maintainer* at this point. I'm making an effort to get more involved, but I don't have the ability to merge pull requests. I think for this `spec` repo it is currently @CraigSellars and @achamely.

I like the idea of releasing and tagging, but I'm trying to keep the Master -> Omni rename a standalone, uncontroversial change.

File: README.md (hunk 3 above)
Comment (msgilligan) [inline README.md]: blank lines removed in aee1b74367e9180800214e995bc774b817ec20d1

**Comment by msgilligan** [comment]
@dacoinminster @CraigSellars @marv-engine @zathras-crypto -- any comments? If not, @CraigSellars will merge these changes on Friday.

**Comment by zathras-crypto** [comment]
No objections from me on the Omni rename :)

I'd like to see the spec get back into a manageable state - there were a bunch of old PRs and outstanding updates to the old spec that never went anywhere so hopefully we can avoid that and keep this one on the new repo clean :)

**Comment by achamely** [comment]
+1 gave it a read through and it looks good to me. Nice job @msgilligan

File: README.md [hunk 4]
```diff
@@ -797,7 +807,7 @@ Say that having created your "Quantum Miner" smart property (which was assigned
 
 This transaction permanently destroys Mastercoins in exchange for favorable placement of this property in the default sort-ordering of properties on every UI. Protocol parsers accumulate all promotions of a property (which can be done by any address which has Mastercoins), with newer promotions being worth more than older promotions. 
 
-To accomplish this time-weighting, a promotion is worth (# Mastercoins spent) * 3^(years since exodus), where "years since exodus" is the number of years (including partial years) since the Mastercoin fund-raiser ended on September 1st 2013, and thus new promotions are always worth 3x as much as year-old promotions and 9x as much as two-year-old promotions if the same number of Mastercoins were spent on each.
+To accomplish this time-weighting, a promotion is worth (# Mastercoins spent) * 3^(years since exodus), where "years since exodus" is the number of years (including partial years) since the Omni fund-raiser ended on September 1st 2013, and thus new promotions are always worth 3x as much as year-old promotions and 9x as much as two-year-old promotions if the same number of Mastercoins were spent on each.
```
Comment (marvgmail) [inline README.md]: minor opinion - I would leave this one as Mastercoin for its historical reference, but it's not a big deal.

**Comment by marvgmail** [comment]
@msgilligan - looks good to me. At some point we should get rid of the manually entered doc version numbers and rely on commit ids or release numbers to identify the version.

File: README.md (hunk 4 above)
Comment (msgilligan) [inline README.md]: Good point, Marv! Fixed.

(Also fundraiser should not be hyphenated, fixed in two places)

**Comment by msgilligan** [comment]
@marvgmail I think the spec could use a major formatting (and content) overhaul/update. It seems to me that revision tracking should be updated at that time, too.

**Comment by msgilligan** [comment]
@CraigSellars @achamely This PR is ready to merge.

**Comment by CraigSellars** [comment]
@msgillian, thank you for the work on this PR

**Comment by msgilligan** [comment]
p.s. Someone with Admin should also change the repository title/description to match the "Omni Protocol Specification" @achamely or @CraigSellars

**Pull request merged**